source/Engine/main.cpp

```diff
@@ -132,10 +134,10 @@ void checkInput() {
 	while ( SDL_PollEvent(&event) ) {
 		switch (event.type) {
 				
-			case SDL_VIDEORESIZE:
-				realWinWidth = event.resize.w;
-				realWinHeight = event.resize.h;
-				setGraphicsWindow(false, true, true);
+			case SDL_WINDOWEVENT_RESIZED:
+				realWinWidth = event.window.data1;
+				realWinHeight = event.window.data2;
+				//setGraphicsWindow(false, true, true);
```

Comment:
I don't remember why I commented it out, probably had some reason :P Needs rechecking.